triple-store/triple-store-impl-jena/src/main/java/com/powsybl/triplestore/impl/jena/TripleStoreJena.java

```diff
@@ -67,6 +71,15 @@ public TripleStoreJena() {
         union = ModelFactory.createDefaultModel();
     }
 
+    @Override
+    public String getImplementationName() {
+        return NAME;
+    }
+
+    public Dataset getDataset() {
+        return dataset;
+    }
```

Comment:
You do not need this getter (encapsulation principle) except if you want to expose internal data outside the class (vulnerability)

Reply:
Fixed